lib/srv/db/cloud/watchers/watcher_test.go

```diff
@@ -44,14 +44,17 @@ func TestWatcher(t *testing.T) {
 	rdsInstance2, _ := makeRDSInstance(t, "instance-2", "us-east-2", map[string]string{"env": "prod"})
 	rdsInstance3, _ := makeRDSInstance(t, "instance-3", "us-east-1", map[string]string{"env": "dev"})
 	rdsInstance4, rdsDatabase4 := makeRDSInstance(t, "instance-4", "us-west-1", nil)
+	rdsInstanceUnavailable, _ := makeRDSInstance(t, "instance-5", "us-west-1", nil, rdsInstanceStatus("stopped"))
```

Comment:
How about adding a test that explicitly verifies the behavior against unrecognized status, like "xxx-this-status-does-not-exist"?

Reply:
added
```
=== RUN   TestWatcher/RDS_unavailable_databases_are_skipped
WARN[0000] Unknown status type: "status-does-not-exist". Assuming RDS instance "instance-5" is available. 
WARN[0000] Unknown status type: "status-does-not-exist". Assuming Aurora cluster "cluster-5" is available. 
=== RUN   TestWatcher/skip_access_denied_errors
=== RUN   TestWatcher/Redshift_labels_matching
=== RUN   TestWatcher/Redshift_unavailable_databases_are_skipped
WARN[0000] Unknown status type: "status-does-not-exist". Assuming Redshift cluster "test" is available. 
```